<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagSpaceFixWidget</class>
 <widget class="QWidget" name="TagSpaceFixWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1174</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tag space tool</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../resources/main_icon.jpg</normaloff>../resources/main_icon.jpg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="fixButton">
       <property name="text">
        <string>Fix tags</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="advancedMode">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Advanced mode</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="tristate">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupTextEdit">
      <property name="title">
       <string>Logs:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupTreeView">
      <property name="title">
       <string>Grouped:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Filter:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="filterLine">
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="expandButton">
           <property name="text">
            <string>&lt;||&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="colapseButton">
           <property name="text">
            <string>|&gt;&lt;|</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeView" name="treeView">
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="libraryPlaceholder">
     <property name="text">
      <string>library path placeholder</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <connections>
  <connection>
   <sender>expandButton</sender>
   <signal>clicked()</signal>
   <receiver>treeView</receiver>
   <slot>expandAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1084</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>1028</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colapseButton</sender>
   <signal>clicked()</signal>
   <receiver>treeView</receiver>
   <slot>collapseAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1126</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>1117</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
